app/src/main/java/com/onboarding/parkingsystemjava/activity/ReserveActivity.java

```diff
@@ -21,7 +22,7 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         this.binding = ActivityNewReservationBinding.inflate(getLayoutInflater());
         setContentView(binding.getRoot());
-        this.presenter = new ReservePresenter(new ReserveView(this, binding), new ReserveModel(ParkingDatabase.getInstance()));
+        this.presenter = new ReservePresenter(new ReserveView(this, binding), new ReserveModel(ParkingDatabase.getInstance(), new ReservationVerifier(ParkingDatabase.getInstance())));
```

Comment:
It's necessary to send this as a parameter?

Reply:
ActivityView requires an Activity in the constructor, and ReserveView extends from ActivityView so i've used **this** sience getActivity() is not aviable

Reply:
hahaha I was asking for the reservation verifier Facu

Reply:
LOL, the verifier is instantiated here for dependency injection reasons, same as the database.